library/platform_util.c

```diff
@@ -133,3 +134,14 @@ struct tm *mbedtls_platform_gmtime_r( const mbedtls_time_t *tt,
 #endif /* _WIN32 && !EFIX64 && !EFI32 */
 }
 #endif /* MBEDTLS_HAVE_TIME_DATE && MBEDTLS_PLATFORM_GMTIME_R_ALT */
+
+#if defined(MBEDTLS_CHECK_PARAMS) && defined(MBEDTLS_DEBUG_INVALID_PARAMS)
+void mbedtls_param_failed( const char* failure_condition, const char* file,
```

Comment:
Style: I think normally the '*' character goes next to the variable name rather than the type.